src/excalidraw-app/index.tsx

```diff
@@ -680,6 +680,13 @@ const ExcalidrawWrapper = () => {
         handleKeyboardGlobally={true}
         onLibraryChange={onLibraryChange}
         autoFocus={true}
+        user={{
+          first_name: "Iron",
+          last_name: "Man",
+          email: "patel.vivek@hackerrank.com",
+          color: "#00ff",
+          // image: "https://lh3.googleusercontent.com/a-/AOh14GhnBcE5G6gu8ZYjEanRJ4-SUNI7GLshrGuJfGoo=s96-c",
+        }}
```

Comment:
**question**: I'm guessing this is just for test? 
Can you elaborate what are we using this for? Is it to determine the current user for new comments?

Reply:
>  I'm guessing this is just for test?

Yes, just for test and it'll not come in the final build of the npm package. (I'll add dummy email and image URL instead of mine)
> Can you elaborate what are we using this for? Is it to determine the current user for new comments?

True, just to determine the current user for new comments! If the image is not present or not valid, then it'll render the user's name's initials and the passed color (in this case #00ff) as a bgcolor for the comment pin